dev/fattest.simplicity/src/componenttest/rules/repeater/EE9PackageReplacementHelper.java

```diff
@@ -35,7 +35,8 @@
 
     private static final Logger Log = Logger.getLogger(EE9PackageReplacementHelper.class.getName());
 
-    public EE9PackageReplacementHelper(String rulesPath){
```

Comment:
you can still keep this old constructor as an option for people, I was just suggesting adding a new ctor with 0-args

Reply:
Added it back in.